.circleci/config.yml

```diff
@@ -23,28 +23,28 @@ defaults: &defaults
         name: Output useful stuff
         command: |
           echo $VERSION > "$CIRCLE_ARTIFACTS/version.txt"
-          # Find the lines of the changelog between releases, escape double quotes, delete empty lines
-          # Simply gets the current changes from top of changelog.
-          #sed -n '{ /------/= }' CHANGES.md \
```

Comment:
Did we previously stick the changelog somewhere as part of the build?

Reply:
Notebooker (which I borrowed the circleci build yaml from) greps out the changes for the new release & includes those in the automated github build/release.  The old arctic travis build never did anything as clever as that, so I removed that part of the notebooker build for arctic